pkg/platform/model/supportedlangs.go

```diff
@@ -0,0 +1,21 @@
+package model
+
+import (
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/pkg/platform/api/mediator"
+	"github.com/ActiveState/cli/pkg/platform/api/mediator/model"
+	"github.com/ActiveState/cli/pkg/platform/api/mediator/request"
+	"github.com/ActiveState/cli/pkg/platform/authentication"
+)
+
+// FetchSupportedLanguages returns the list of languages that the Platform supports ATM
+func FetchSupportedLanguages(auth *authentication.Auth) ([]string, error) {
```

Comment:
Is there a technical reason for requiring auth? I can't think of any reason why we should need auth for this.

Reply:
Just future compatibility.  I removed it for now.